Source/WebCore/page/NavigationHistoryEntry.h

```diff
@@ -85,6 +85,8 @@ class NavigationHistoryEntry final : public RefCounted<NavigationHistoryEntry>,
     void derefEventTarget() final { deref(); }
     void eventListenersDidChange() final;
 
+    RefPtr<ScriptExecutionContext> protectedScriptExecutionContext() const { return scriptExecutionContext(); }
```

Comment:
We shouldn't need this we subclass ActiveDOMObject which exposes `protectedScriptExecutionContext()` already.

Reply:
True-fixed :)